src/scripts/main.js

```diff
@@ -1,5 +1,20 @@
 'use strict';
 
 document.addEventListener('click', (e) => {
-  // write code here
+  const wall = document.querySelector('.wall');
+  const spider = wall.querySelector('img');
+
+  const wallRect = wall.getBoundingClientRect();
+  const spiderWidth = spider.offsetWidth;
+  const spiderHeight = spider.offsetHeight;
+
+  let x = e.clientX - 5 - wallRect.left - spiderWidth / 2;
```

Comment:
The calculation for the x-coordinate includes a subtraction of 5, which seems arbitrary. Ensure this value is intentional and necessary for your design.